### PR TITLE
Modified Readme file

### PR DESCRIPTION
**Pull request by Hilyas68**
I have added the doc for Article for review

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,45 +19,78 @@ If there is an API endpoint that isn't currently supported, feel free to open an
...
-```csharp
-using Forem.Api;
+## Usage
 
-public class MyClass
-{
-	private IArticlesService ArticlesService { get; }
-	private ITagsService TagsService { get; }
+- For Some of the methods, DEV API key `apiKey`  is required to be passed to authenticate you as a user. 
+  To Obtain one, check the authentication section [GET API KEY](https://docs.dev.to/api/#section/Authentication)
```
Comment (Turnerj) [inline readme.md:31]: Nit: Lowercase "o" for "obtain"

File: readme.md [hunk 2]
```diff
@@ -19,45 +19,78 @@ If there is an API endpoint that isn't currently supported, feel free to open an
...
-		// Your code here...
-	}
-}
+		return provider.GetRequiredService<IHttpClientFactory>().CreateClient(string.Empty);
+	});
+
 ```
 
-### Without DI
+Havng all this setup, then you're good to go!!
```
Comment (Turnerj) [inline readme.md:47]: Nit: Missing "i" in "Having"

File: readme.md [hunk 3]
```diff
@@ -19,45 +19,78 @@ If there is an API endpoint that isn't currently supported, feel free to open an
...
+        }
+
+```
 
-var articlesService = new ArticlesService(new Uri("https://dev.to/"), new HttpClient());
-var tagsService = new TagsService(new Uri("https://dev.to/"), new HttpClient());
+To get all articles you simply just call `GetArticlesAsync` passing optional params `page` and `perPage` with default values `1` & `30` respectively.
+
+```csharp
+	[HttpGet("all")]
```
Comment (Turnerj) [inline readme.md:80]: Nit: Indent this to line up with the code below

File: readme.md [hunk 4]
```diff
@@ -19,45 +19,78 @@ If there is an API endpoint that isn't currently supported, feel free to open an
...
+
+            articles = await _articlesService.GetArticlesAsync(page, itemPerPage);
+            
+            return Ok(articles);
+        }
 
-var articles = await articlesService.GetArticlesAsync();
 ```
+
+Feel free to check out other methods/endpoints [Aticles Doc](https://docs.dev.to/api/#operation/getArticles)
```
Comment (Turnerj) [inline readme.md:95]: Nit: Typo in "Articles", missing the "r"

**Review by Turnerj — COMMENTED** [review]
Nice work, I like it! I've left a few comments inline where some minor corrections are needed.

File: readme.md (hunk 4 above)
Comment (Hilyas68) [reply readme.md:95]: I will make the corrections. Thanks

**Comment by Hilyas68** [comment]
I have pushed an update

**Review by Turnerj — APPROVED** [review]

**Pull request merged**